docs/Project.toml

```diff
@@ -8,4 +8,5 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 
 [compat]
+DataFrames = "~1.3"
```

Comment:
Lock in the DataFrames version in docs so that the pretty printing remains the same.

Reply:
I'm not sure I like this. Better get failures if printing changes so that we can keep the docs up to date? Otherwise we'll likely forget about that.

Reply:
I agree with your philosophy but we hadn't been failing on doctests even though that behavior changed a while back.  I don't mind getting rid of the compat restriction, but we need to be better about keeping the doctests up to date.

Reply:
Indeed. We should pass `strict=true`, right?

Reply:
Uhh.... yes. I'll update that too.

Reply:
@nalimilan done